<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="sf-cpu-729" failures="0" tests="1" name="com.TestRails.TestCases.passmain" time="25.932" errors="1" timestamp="2022-07-08T14:49:11 IST" skipped="0">
  <testcase classname="com.TestRails.TestCases.passmain" name="passmain" time="25.932">
    <error message="Compound class names not permitted
For documentation on this error, please visit: https://selenium.dev/exceptions/#invalid_selector_exception
Build info: version: &#039;4.2.2&#039;, revision: &#039;683ccb65d6&#039;
System info: host: &#039;sf-cpu-729&#039;, ip: &#039;fe80:0:0:0:c5a:2d33:f61d:42a3%en0&#039;, os.name: &#039;Mac OS X&#039;, os.arch: &#039;x86_64&#039;, os.version: &#039;11.6.2&#039;, java.version: &#039;18.0.1.1&#039;
Driver info: driver.version: unknown" type="org.openqa.selenium.InvalidSelectorException">
      <![CDATA[org.openqa.selenium.InvalidSelectorException: Compound class names not permitted
For documentation on this error, please visit: https://selenium.dev/exceptions/#invalid_selector_exception
Build info: version: '4.2.2', revision: '683ccb65d6'
System info: host: 'sf-cpu-729', ip: 'fe80:0:0:0:c5a:2d33:f61d:42a3%en0', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '11.6.2', java.version: '18.0.1.1'
Driver info: driver.version: unknown
at org.openqa.selenium.By$ByClassName.<init>(By.java:310)
at org.openqa.selenium.By.className(By.java:100)
at com.TestRails.Pages.passed.pass(passed.java:66)
at com.TestRails.TestCases.passmain.passmain(passmain.java:41)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- passmain -->
  <system-out/>
</testsuite> <!-- com.TestRails.TestCases.passmain -->
